tests/unit/Wells.spec.js

```diff
@@ -31,6 +31,9 @@ describe('Wells', () => {
       cmp = mount(Wells, {
         mixins: [WellProperties],
         props: data,
+        render() {
+          return null
+        },
```

Comment:
I hope we do not intend to test this component apart from the behaviour defined in props and WellProperties.